docs/src/main/asciidoc/spring-di.adoc

```diff
@@ -48,33 +122,54 @@ mvn io.quarkus:quarkus-maven-plugin:{quarkus-version}:create \
 cd spring-di-quickstart
 ----
 
-This command generates a Maven project with a REST endpoint and imports the `spring-di` extension.
-
-If you already have your Quarkus project configured, you can add the `spring-di` extension
-to your project by running the following command in your project base directory:
+* If you already have a {BrandName} project you can add the `quarkus-spring-di` extension to it using the command line:
+--
+. Navigate to the the top-level directory of your {BrandName} Maven project:
++
+[source,bash,options="nowrap",subs="+quotes,attributes+"]
+----
+cd <project_directory>
+----
 
+. Enter the following command to add the `quarkus-spring-di` extension to the `pom.xml` file of your project:
++
 [source,bash]
 ----
 ./mvnw quarkus:add-extension -Dextensions="spring-di"
 ----
-
-This will add the following to your `pom.xml`:
-
-[source,xml]
++
+This adds the following dependency to your `pom.xml` file:
++
+[source,xml,options="nowrap",subs="+quotes,attributes+"s]
 ----
 <dependency>
     <groupId>io.quarkus</groupId>
     <artifactId>quarkus-spring-di</artifactId>
 </dependency>
 ----
+--
```

Comment:
Looks like an unnecessary line here `--` shouldn't be there, I think.

Reply:
@oraNod It's not entirely unnecessary, it is more or a workaround for a specific downstream issue.  This step contains an ordered list inside an unordered list item that does not render correctly if not wrapped in the open block `--` markup in the downstream. If the open block wasn't there, the ordered list would automatically be rendered as an unordered list.

Reply:
OK, I see that. So it's for an "either/or" procedure. In this case it seems like a different procedure to me. Maybe consider making them a little more distinct by introducing them with "To do x" after the `.Procedure` tag.

"To create a new Maven project for the Spring Dependency Injection example application, enter the following commands:"

Also note that `$` should really go before each command (`$ mvn ...` and `$cd ...`) to make it clear that there are two separate commands. 

"To add the `quarkus-spring-di` extension to an existing project, do the following:"

When I view the rendered HTML as it is now it looks like the 2nd bullet is part of the first procedure, but it's actually two separate procedures. 

Also, it seems to me others could make the same mistake that I did when editing and remove the trailing `--` characters, which will hose downstream.

Reply:
@oraNod OK I see your point. This procedure is an either/or use case, indeed.
In either case you are setting up a project, to be able to continue with the ensuing steps.
I had formatted each case as a separate procedure but it seemed even less clear that way.
I'll take your advice and try to provide clearer lead-in sentence for each of the 2 alternatives.  

I can drop in a comment asking contributors not to remove the open block. Do you think that would be helpful?

About the PS prompt, I agree with your reasoning, but this is not standard practice in the upstream Quarkus docs at this point, and it makes it slightly harder to copy and paste the code from the examples
@maxandersen would it be OK to add shell prompts to command examples or are there other considerations that I might be unaware of?